### PR TITLE
Migrate etcd, master, and node roles for #275.

### PR DESCRIPTION
**Pull request by davidewatson**

### CODE REVIEW

File: ansible/roles/kraken.master/kraken.master.docker/templates/units.kraken-apiserver-ssl.part.jinja2 [hunk 1]
```diff
@@ -18,8 +18,8 @@
      --mount volume=input,target=/input \
      --mount volume=output,target=/output \
      --set-env=HOST_IP=${DEFAULT_IPV4} \
-     --set-env=KUBE_SERVICE_IP={{ kraken_config.serviceCidr | ipaddr('1') | ipaddr('address') }} \
-     --set-env=SERVICE_DNS_NAME=apiserver.{{kraken_config.cluster}}.internal \
-     quay.io/samsung_cnct/drunkensmee:v0.5 \
+     --set-env=KUBE_SERVICE_IP={{ cluster.network | ipaddr('1') | ipaddr('address') }} \
+     --set-env=SERVICE_DNS_NAME=apiserver.{{ cluster.name }}.internal \
+     quay.io/samsung_cnct/drunkensmee:v0.4 \
```
Comment (coffeepac) [inline ansible/roles/kraken.master/kraken.master.docker/templates/units.kraken-apiserver-ssl.part.jinja2]: this looks like a gitting artifact.  we should be using v0.5.  This should also be a config parameter but that's for another time.

Comment (davidewatson) [inline ansible/roles/kraken.master/kraken.master.docker/templates/units.kraken-apiserver-ssl.part.jinja2]: Good catch.

**Comment by leahnp** [comment]
Any review instructions?

**Comment by davidewatson** [comment]
Since we will be developing against the k2recon branch until all stanzas have been migrated, anyone with a migration ticket will need checkout that branch, generate a new config, and then test that they can bring a cluster up with it. That will not test this particular code until it is merged, but will be necessary once it is merged.

**Pull request merged**